File: 2.1/ICS_2204-programming_languages/0x00-python_assignments/bmi_calculator.py
## SCT211-0009/2022
## Njoroge Kanyagia

def bmicalculator(h, w):
    bmi = w / (h ** 2)
    
    print(f"Your BMI is: {bmi}")
    
    if bmi < 18.5:
        print("You are underweight!")
    elif 18.5 <= bmi <= 24.9:
        print("You have normal weight!")
    elif 25 <= bmi <= 29.9:
        print("You are overweight!")
    else:
        print("You are obese!")

def main():
    weight = float(input("Enter your weight (in kilograms): "))
    height = float(input("Enter your height (in meters): "))
    
    print()
    bmicalculator(height, weight)

if __name__ == "__main__":
    main()

File: 2.1/ICS_2201-OOP_II/bmi_calculator.java
// SCT211-0009/2022
// Njoroge Kanyagia

import java.util.Scanner;

public class bmi_calculator {
    public static void calculateBMI(double h, double w){
        double bmi = (w / (h * h));
        
        System.out.printf("Your BMI is : %f\n", bmi);
        
        if (bmi < 18.5){
            System.out.println("You are underweight!");
        }else if (bmi <= 24.9){
            System.out.println("You have normal weight");
        }else if (bmi <= 29.9){
            System.out.println("You are overweight!");
        }else{
            System.out.println("You are obese!");
        }
    }
    public static void main(String[] args){
        double weight, height;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter your weight in kilograms: ");
        weight = scanner.nextDouble();
        
        System.out.print("Enter your height in metres: ");
        height = scanner.nextDouble();      
        
        scanner.close();

        calculateBMI(height, weight);
    }
}